deployments/deploy-ssh.sh

```diff
@@ -509,6 +529,12 @@ spec:
       value: "${VSPHERE_PWD}"
     - name: VSPHERE_HOST_IP
       value: "${VSPHERE_HOST_IP}"
+    - name: CONTROL_PLANE_URL
+      value: "${CONTROL_PLANE_URL}"
+    - name: CONTROL_PLANE_KUBECONFIG
+      value: "/tmp/ctrl_kubeconfig"
```

Comment:
This is not used anymore, we should remove this and just rely on URL? 
Also remove the hard coding, if the path does not change, please create a variable for this.